registry.json

```diff
@@ -2115,5 +2115,20 @@
     "telegram":"t0m3kf",
     "operatorName":"",
     "github":"t0m3kf"
+  },
+   {
+    "ticker":"NORB",
+    "poolId":"71b00c23502112d4c634e325f28c9ade2a4ba4ec251677d306ae6b7b",
+    "social":{
+      "facebook":"",
+      "twitter":"NorbPool",
```

Comment:
Hey. how about we get some action on twitter. Just to make sure a real person is behind it.

Reply:
Hey, I have made a post and added some information for location and bio. I will continue to add new posts. Thank you!

Reply:
I just released my first video on my YouTube channel if you want to check it out. I am looking forward to joining the alliance 😀!